### DISCUSSION TITLE
Stacked items

### DISCUSSION THREAD

**Discussion by thedoofus** [ismail9k/vue3-carousel#329] [Ideas]
I'm not sure if this is a ridiculous idea or not but it's something I'd quite like to do on a project I'm currently working on.
I know there's breakpoints that I can use, but I can't find a way to "stack" slides like I've demonstrated in the image below. 

![image](https://github.com/ismail9k/vue3-carousel/assets/5920857/c3a7093f-e95f-4200-91a5-9e03adb45945)

The reason for this is that on mobile or smalls screens I often want to show more than one item at a time vertically, because there's not really enough space to show more than one slide on a row. So it would be ideal if I could show a column of items, rather than a row. The carousel would still _rotate_ horizontally.

Hope that all makes sense - is this possible already, or is it a feature worth adding?

**Comment by ismail9k** [comment]
At present, this feature is not available. However, it would be worth considering its inclusion in the future.
